source/agora/consensus/data/Block.d

```diff
@@ -150,6 +150,34 @@ public struct Block
     public Hash[] merkle_tree;
 
 
+    /***************************************************************************
+
+        Update the Block header with new signature
+
+        Params:
+            block = block to be signed
+            signature = new signature
+            validators = mask to indicate who has signed
+
+    ***************************************************************************/
+
+    public Block updateSignature (const Signature signature,
```

Comment:
This ended up in the wrong commit?

The commit title is `Add signature and validators to pretty print`